compiler/src/dotty/tools/dotc/core/Flags.scala

```diff
@@ -532,8 +532,17 @@ object Flags {
   /** Flags that can apply to a module class */
   val RetainedModuleClassFlags: FlagSet = RetainedModuleValAndClassFlags | Enum
 
-  /** Flags retained in export forwarders */
-  val RetainedExportFlags = Given | Implicit | Inline | Transparent
+  /** Flags retained in term export forwarders */
+  val RetainedExportTermFlags = Infix | Given | Implicit | Inline | Transparent | Erased | HasDefaultParams | NoDefaultParams | ExtensionMethod
+  // TODO: SHOULD WE ADD: ConstructorProxy ?
```

Comment:
I don't think so: a method that calls a constructor proxy is not itself a constructor proxy.

Reply:
I will drop the todos